src/layers/models/layersManager.ts

```diff
@@ -342,9 +342,10 @@ export class LayersManager {
       isCleaned: false,
       shouldReturnTasks: false,
     };
+    const ingestionJobTypes = this.config.get<string[]>('forbiddenTypesForParallelIngesion');
     const jobs = await this.db.getJobs<Record<string, unknown>, ITaskParameters | IMergeTaskParams>(findJobParameters);
     jobs.forEach((job) => {
-      if (job.status == OperationStatus.IN_PROGRESS || job.status == OperationStatus.PENDING) {
+      if ((job.status == OperationStatus.IN_PROGRESS || job.status == OperationStatus.PENDING) && ingestionJobTypes.includes(job.type)) {
         const message = `Layer id: ${productId} product type: ${productType}, job is already running`;
```

Comment:
`Layer id: ${productId} product type: ${productType}, conflicting job ${ingestionJobTypes[0]} is already running`